spec/unit/provider/service/launchd_spec.rb

```diff
@@ -147,21 +198,45 @@
     end
   end
 
-  describe "when enabling the service" do
+  [[10, "10.6"], [13, "10.9"]].each do |kernel, version|
```

Comment:
https://github.com/puppetlabs/puppet/commit/efd2a45646eaedd3aa744ebcf7beb94739653db5 specifically removes support for <= 10.6, so I could change this to test 10.7. I wanted to test the range of what we seem to support.